docs/pages/dev-guide/developer-guide.md

```diff
@@ -1228,7 +1228,9 @@ The path is relative to the sandbox path if not preceded by a leading "/". The s
 
 ### Proxy
 
-The proxy allows one to expose more than one endpoint through adminrouter. It is only supported on DC/OS 1.9 clusters.
+The proxy allows one to expose more than one endpoint through adminrouter. It is only supported on DC/OS 1.9 clusters. An example of this can be found in the proxylite framework in this repo. The proxylite framework itself implements this properly.
```

Comment:
Suggest: "The proxy allows you to expose more than one endpoint through Admin Router. The proxy is only supported on DC/OS 1.9 Clusters. An example of a correct proxy implementation can be found in the ProxyLite framework in this repository."